es6/lib/utils.js

```diff
@@ -272,7 +272,7 @@ var range = function(max) {
 };
 
 var hasKey = function(obj, key) {
-  return typeof obj[key] !== undefined;
+  return typeof obj[key] !== 'undefined';
```

Comment:
I thought about it but I really cannot see a good reason why you would want your experiment parameter to be `undefined`. Maybe so that the control variation and the non experiment member group react on the same `undefined` return value in the logic.
I don't think that should be encouraged though. In my opinion parameter values should always be very explicit.